src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -31,11 +33,10 @@ The high-level steps in this tutorial are:
 
 ## Before you begin
 
-- Two separate CockroachDB clusters (primary and standby) with a minimum of three nodes each, and each using the same CockroachDB {{page.version.version}} version.
+- Two separate CockroachDB clusters (primary and standby) with a minimum of three nodes each, and each using the same CockroachDB {{page.version.version}} version. The standby cluster should be the same version or one version ahead of the primary cluster.
```

Comment:
Added this sentence to match the current description in the "Cluster versions and upgrades" section.